cmd/package-operator-manager/components/components.go

```diff
@@ -98,15 +98,13 @@ func ProvideManager(
 		Scheme:                     scheme,
 		MetricsBindAddress:         opts.MetricsAddr,
 		HealthProbeBindAddress:     opts.ProbeAddr,
-		Port:                       9443,
+		WebhookServer:              webhook.NewServer(webhook.Options{}),
```

Comment:
Where did the hardcoded port go?

Reply:
The `Port` field in [manager#Options](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.15.0/pkg/manager#Options) is deprectated, there is a comment that says to use a webhook server instead. The value `9443` just happens to be the default in [webhook#Options](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.15.1/pkg/webhook#Options), so I left it out.

Reply:
Alrighty got it. :)
I'd tend to keep passing 9443 to avoid the default changing under our noses but both options are 100% ok :)

Reply:
Yep, that makes sense